viam-cartographer/src/carto_facade/carto_facade_test.cc

```diff
@@ -725,6 +760,17 @@ BOOST_AUTO_TEST_CASE(CartoFacade_demo) {
                    VIAM_CARTO_SUCCESS);
     }
 
+    {
+        viam_carto_get_internal_state_response isr;
+        BOOST_TEST(viam_carto_get_internal_state(vc, &isr) ==
+                   VIAM_CARTO_SUCCESS);
+        BOOST_TEST(blength(isr.internal_state) >
+                   last_internal_state_response_size);
```

Comment:
[q] Does the internal state always increase in size?

If you don't know off the top of your head, I'm fine assuming yes.

Reply:
It appears to always increase (at least for 3 lidar readings) in size on linux, on mac it does not. I'm not sure why.